lib/styleDictionary/util.js

```diff
@@ -0,0 +1,98 @@
+const DEFAULT_KEY = 'default';
+
+/**
+ * Simple utility function to either wrap the given output object in an object
+ * as a prop with key 'propKey', or return an empty object. Usefull to create
+ * the desired CTI output of Style Dictionary.
+ * 
+ * @param String propKey: key in which the output should be wrapped 
+ * @param Object obj: object to wrap 
+ */
+const outputHelper = (propKey, obj) => {
+  const output = {};
+
+  if (Object.keys(obj).length) {
```

Comment:
What is this check used for? I can see two cases:
	- trying to check if obj is actually an object, in which case code would crash on falsy values
	- trying to see if obj has properties defined.

If desired result is the latter, please do not rely on 0 being falsy in JS, but be explicit instead by checking if length is gt 0

```suggestion
  if (Object.keys(obj).length > 0) {
```